src/components/charts/NetWorthHistogram.tsx

```diff
@@ -97,94 +97,97 @@ export default function NetWorthHistogram({
-                callback: (value) => moneyToString(value as number, unit),
-              },
+            maintainAspectRatio: false,
+            interaction: {
+              mode: 'index',
             },
-          },
-          plugins: {
-            title: {
-              display: true,
-              text: 'Net Worth',
-              align: 'start',
-              padding: {
-                top: 0,
-                bottom: 30,
+            scales: {
+              x: {
+                stacked: true,
+                type: 'time',
+                time: {
+                  unit: 'month',
+                  displayFormats: {
+                    month: 'MMM-yy',
+                  },
+                },
+                grid: {
+                  display: false,
+                },
+                ticks: {
+                  align: 'center',
+                },
               },
-              font: {
-                size: 18,
+              y: {
+                grace: 1,
+                beginAtZero: false,
+                position: 'left',
+                border: {
+                  display: false,
+                },
+                ticks: {
+                  maxTicksLimit: 10,
+                  callback: (value) => moneyToString(value as number, unit),
+                },
               },
             },
-            legend: {
-              display: showLegend,
-              position: 'bottom',
-              onClick: () => {},
-              labels: {
-                usePointStyle: true,
-                pointStyle: 'circle',
-                boxHeight: 8,
-                boxWidth: 8,
+            plugins: {
+              title: {
+                display: true,
+                text: 'Net Worth',
+                align: 'start',
+                padding: {
+                  top: 0,
+                  bottom: 30,
+                },
+                font: {
+                  size: 18,
+                },
               },
-            },
-            tooltip: {
-              backgroundColor: '#323b44',
-              callbacks: {
-                label: (ctx) => `${ctx.dataset.label}: ${moneyToString(Number(ctx.raw), unit)}`,
-                labelColor: (ctx) => ({
-                  borderColor: '#323b44',
-                  backgroundColor: ctx.dataset.backgroundColor as string,
-                  borderWidth: 3,
-                  borderRadius: 2,
-                }),
+              legend: {
+                display: showLegend,
+                position: 'bottom',
+                onClick: () => {},
+                labels: {
+                  usePointStyle: true,
+                  pointStyle: 'circle',
+                  boxHeight: 8,
+                  boxWidth: 8,
+                },
+              },
+              tooltip: {
+                backgroundColor: '#323b44',
+                callbacks: {
+                  title: (tooltipItem) => labels[tooltipItem[0].dataIndex].toFormat('DD'),
```

Comment:
we now display the day the data is computed to the user in this chart so they it gives clarity